Docs: Update docs for image (#7369)
slint-ui/slint: docs/astro/src/content/docs/reference/primitive-types.mdx

Loading image from `http` is only supported in [SlintPad](https://slintpad.com).

Supported format are SVG, as well as formats supported by the [`image` crate](https://crates.io/crates/image):
AVIF, BMP, DDS, Farbfeld, GIF, HDR, ICO, JPEG, EXR, PNG, PNM, QOI, TGA, TIFF, WebP.

<Tabs syncKey="dev-language">
<TabItem label="C++">
In C++, properties or struct fields of the image type are mapped to [`slint::Image`](https://docs.slint.dev/latest/docs/cpp/api/structslint_1_1Image).
</TabItem>
<TabItem label="Rust">
In Rust, properties or struct fields of the image type are mapped to [`slint::Image`](https://docs.slint.dev/latest/docs/rust/slint/struct.Image).

> Some format might be disabled by not enabling the `image-default-formats` cargo feature which is enabled by default.

Reviewer: This is a double-negative ("might be disabled by not enabling"). I suggest to simplify this:

```suggestion
Enable the `image-default-formats` feature if you're loading images other than PNG and JPG.
```

Author: Adding SVG and a note that it is the default:

```suggestion
Enable the `image-default-formats` feature (enabled by default) if you're loading images other than PNG, JPEG, and SVG.
```

> Enable the `image-default-formats` feature (enabled by default) if you're loading images other than PNG, JPEG, and SVG.

Reviewer: Ah, I keep forgetting about it being default - somehow I thought it isn't. I think advice to enable something that's enabled by default is of no use. I'd rather omit this sentence and we have an "optimzation" section (how to reduce the binary size, speed up compilation), where we can explain that this could be disabled. But in the most prominent place here I think it's confusing.

```suggestion
```
